### ISSUE TITLE
maybe include (optional) links in taxonomy/lineage databases?

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#1969]
it sure would be nice to have a way to go from a specific taxonomy entry (for a species) to a Web URL with more information about that species.

One possible way to do this would be to include URLs in the spreadsheets, and make them available in sourmash tax output.

For NCBI, we could construct these from the NCBI taxid.

For GTDB, I suspect there is a straightforward linking approach or scheme.

For LINS/genomerxiv, we're talking about how to do this now :).

**Comment by ctb** [comment]
would be in aid of https://github.com/sourmash-bio/sourmash/issues/265

**Comment by mr-eyes** [comment]
Where should this information go/be produced?
Is it something that would be integrated into the taxonomy files provided for `sourmash tax` ?

**Comment by ctb** [comment]
> Is it something that would be integrated into the taxonomy files provided for `sourmash tax` ?

yes, I think so. Unless we find that there is a surprising amount of consistency around how to call out to ncbi, genomerxiv, and gtdb web sites ;)
